src/core/util/util.js

```diff
@@ -739,6 +739,17 @@ function UtilFactory($document, $timeout, $compile, $rootScope, $$mdAnimate, $in
       return current;
     },
 
+    /**
+     * Parses an attribute value to false if the value is falsy but not an empty value
+     * otherwise it will be true.
+     * Will be false if
+     * @param value Interpolated Attribute Value
+     * @returns {boolean}
+     */
+    parseAttributeBoolean: function(value) {
```

Comment:
@DevVersion Can you remove the check for '0' and update the PR?  Other than that, it looks good to me.